app/mailers/user_mailer.rb

```diff
@@ -81,7 +81,7 @@ def survey(user)
 													password: ENV.fetch('SURVEY_EMAIL_PASSWORD'),
 													address: ENV.fetch('SURVEY_EMAIL_ADDRESS'),
 													port: ENV.fetch('SURVEY_EMAIL_PORT') }
-		mail(:subject => "openSNP: A one-question survey on height", :to => @user.email,:from => "survey@opensnp.org",delivery_method_options: delivery_options)
+		mail(:subject => "openSNP: Read our survey results and meet Open Humans", :to => @user.email,:from => "survey@opensnp.org" ,delivery_method_options: delivery_options)
```

Comment:
Some time would be cool. @philippbayer and I wanted to work on it a lot during those two days if possible. And hopefully other people chime in as well!